src/shipit_code_coverage/shipit_code_coverage/github.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, cache_root, client_id, access_token):
         self.cache_root = cache_root
         self.gecko_dev_user = secrets.get(secrets.GECKO_DEV_USER)
         self.gecko_dev_pwd = secrets.get(secrets.GECKO_DEV_PWD)
+        self.hg_git_mapper = secrets[secrets.HG_GIT_MAPPER] if secrets.HG_GIT_MAPPER in secrets else 'https://api.pub.build.mozilla.org/mapper'
```

Comment:
lets replace the default with `https://mapper.mozilla-releng.net` since that url will be gone with end of july